.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Release Check List
+about: v<TARGET_RELEASE>
+title: "v<TARGET_RELEASE Check List"
+---
+
+## v<TARGET_RELEASE>
+
+* [ ] - Get the latest ocicrypt-rs used as part of image-rs
```

Comment:
Sorry, just to clarify - by `Get the latest ocicrypt-rs used as part of image-rs` do you mean identify the commit of ocicrypt-rs that is used by the version of image-rs that the kata-agent is pulling in, or something else - like bump the version of ocicrypt-rs that image-rs is using to the latest ocicrypt-rs sha (and then test it)?

Reply:
Is it also worth for these mentioned where/how they are pulled in, e.g. in Cargo.toml in this case to point any less experienced people in the right direction?